start.js

```diff
@@ -0,0 +1,74 @@
+const http = require("http");
+const soldierRoute = require("./routes/soldiers.js");
+const dutiesRoute = require("./routes/duties");
+const justiceBoardRoute = require("./routes/justiceBoard");
+const configs = require('./configFile');
+const MongoClient = require('mongodb').MongoClient;
+
+// Connection URL
+const mongoUrl = configs.dbUrl;
+
+// Database Name
+const dbName = configs.dbName;
+
+function routesCases(res, req, primeFolder, numFolders, db, method, url, body, resultCallback) {
+  if (/soldiers.*/.test(primeFolder) && numFolders <= 3) {
```

Comment:
i commented on this allready. use switch case instead. this is too many branchings and looks messy